### DISCUSSION TITLE
osu! or osu!lazer on Microsoft Store as UWP

### DISCUSSION THREAD

**Discussion by Likver** [ppy/osu#23600] [Ideas] [answered]
So ive been wondering for a while now, would it be possible for the devs to port osu! (or lazer) as UWP and upload the game to the MS store? The UWP part its because of XBOX consoles, they can run UWP files, which Windows PCs also can, so basically im asking for "is there any chance, osu! or osu!lazer will be available for XBOX consoles in the future?" It would be amazing if u made this, if not, its totally understandable, thanks anyways!

**Comment by Jed666** [comment]
this idea could also be applied to the mac app store too prob.

**Comment by Jed666** [comment]
tho i doubt there would be many people downloading osu from those stores when they can just get it from the website, and i dont think its practical to play the rulesets unless you have a special peripheral like a tablet/mouse and/or keyboard. etc. osu!catch is the only viable option for a standard controller and its sadly not a popular gamemode among the osu! community.

**Comment by ItsShamed** [comment]
IIRC with .NET this also means dropping seamless cross-platform support (remember osu!lazer also supports Linux)
So personnally I would be against it

**Comment by peppy** [comment, marked as answer]
See https://github.com/ppy/osu/discussions/18074#discussioncomment-5055332, roughly the same reasoning / flow for whether we would ever support this.
